app/models/user.rb

```diff
@@ -24,5 +24,8 @@ class User < ApplicationRecord
 
   has_many :event_invites
 
+  def token_validation_response
```

Comment:
it doesn't look like a good place to call serializer here. why do we do this here?